recipes/pangolin/meta.yaml

```diff
@@ -19,23 +19,23 @@ requirements:
     - pip
   run:
     - python >=3.7
-    - biopython ==1.74
+    - biopython >=1.74
     - pandas >=1.0.1
-    - tabulate
     - joblib >=0.11
-    - scikit-learn <1.3.0
+    - scikit-learn >=0.23.1
     - pulp >=2
     - minimap2 >=2.16
-    - snakemake-minimal >=5.13,<=7.30.1
+    - snakemake-minimal >=5.13,!=7.30.1
     - gofasta
-    - usher >=0.6.2
+    - usher >=0.5.4
     - ucsc-fatovcf >=426
     - git
     - git-lfs
     - scorpio >=0.3.12
     - constellations >=0.0.15
     - pangolin-data >=1.2.133.2
     - conda
+    - wheel >=0.34
```

Comment:
why is this needed?

Reply:
It's in setup.py 🤷 
https://github.com/cov-lineages/pangolin/blob/master/setup.py

Reply:
conda-forge pip brings in wheel as a dependency so there's no need for bioconda to specify it again.